documentatie/flowcharts/data-entry-forms.md

```diff
@@ -18,13 +18,13 @@ flowchart TD
     %% elements
     flow-start([start])
 
-    disabled-styling([disabled styling])
-    current-styling([current styling])
-    unsubmitted-styling([unsubmitted styling])
-    error-styling([error styling])
-    warning-styling([warning styling])
-    empty-styling([empty styling])
-    green-check-styling([valid styling])
+    styling-disabled([styling: disabled])
+    styling-current([styling: current])
+    styling-unsubmitted([styling: unsubmitted])
```

Comment:
should this be: 'not-validated' in stead of 'unsubmitted'? Not sure how/where we are going to save/cache this data, but if we do that in the database, we do submit the data, we just don't validate it yet.

Reply:
Good point, let's update the flowchart when we start to save/cache the data. Because right now I don't think that state is even reachable, since we either submit + validate or we discard.